lib/coins/nvc.js

```diff
@@ -0,0 +1,47 @@
+/*
+  info from:
+  https://github.com/novacoin-project/novacoin
+  main.cpp: see unsigned char pchMessageStart[4]
+  base58.h: see unnamed enum
+*/
+
+var assign = require('object-assign')
+
+var common = {
+  name: 'Novacoin',
+  unit: 'NVC'
+}
+
+var main = assign({}, {
+  port: 7777,
+  portRpc: 8344,
+  protocol: {
+    magic: 0xe4e8e9e5 // careful, sent over wire as little endian
```

Comment:
As I understand bytes should be reversed https://github.com/novacoin-project/novacoin/blob/db0f9d16fab4183f6156f2ac652526cf22d70dd7/src/main.cpp#L3128